packages/json-rpc-middleware-stream/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@metamask/json-rpc-middleware-stream",
+  "version": "6.0.0",
```

Comment:
Hmm, I'm not sure we should bump the version yet. Typically we only bump packages in a release PR. Given that other teams can create release PRs whenever they want to, if we do this now, this new version may be published without our knowledge. What do you think about waiting to do this? I believe the change to `changelog:validate` can stay the same though (unless I'm missing something).

Reply:
This one's a little tricky. Usually the first post-migration release should happen in core as the last step in the migration checklist, but in this case the complication is that `json-rpc-middleware-stream` has never been renamed into the `@metamask/` namespace. 

I think ideally the rename and release should happen in the original repo before git history migration (I'll add an entry about this in the checklist). However, in this case, that essentially means restarting the migration. I don't think doing the rename+release at this stage is safe or desirable, though. 

We could do it as part of the first post-migration release in core. I'll have to think about whether that could introduce any issues.

Reply:
One issue of doing the rename post-migration is that yarn constraints requires all subpackages to have `@metamask/`-namespaced "name" fields in their package.json, so we can't merge this PR without this change.

Reply:
Ah sorry, I forgot that we were doing a rename here. Well, we'll just have to keep an eye on releases then, no problem.